src/Api/Filter/JeMengageSurveyScopeFilter.php

```diff
@@ -49,6 +49,18 @@ protected function applyFilter(
                     ->setParameter('zones', $this->zoneRepository->findForJecouteByReferentTags($user->getManagedArea()->getTags()->toArray()))
                 ;
                 break;
+            case ScopeEnum::CORRESPONDENT:
+                $queryBuilder
+                    ->leftJoin(LocalSurvey::class, 'ls', Join::WITH, sprintf('ls.id = %s.id', $alias))
+                    ->orWhere((new Orx())
+                        ->add(sprintf('%s INSTANCE OF :national', $alias))
+                        ->add(sprintf('%s INSTANCE OF :local AND ls.zone = :zone', $alias))
```

Comment:
A correspondent managing Paris (as department) could not see surveys, because they are linked to a borough.
Should we check also children zones of managed zone?